docs/deposits/index.adoc

```diff
@@ -156,30 +218,40 @@ the slice, and that the new difficulty conforms to Bitcoin's adjustment
 algorithm. In other words, our light relay tracks only Bitcoin's current
 difficulty, and no other information about its state.
 
-Knowing the current difficulty gives stateless SPV proofs stronger recency
-assurances. Any newly-generated stateless SPV must include that difficulty in
-its header chain. And that difficulty is not known to any party in advance.
-Miners with an `n`-fraction (as usual, `n >= 2` due to the 51% assumption) of
-the hashrate have a `1/n` chance of being allowed to set the difficulty, and
-thus have a `1/n` chance of being able to successfully predict it 2 weeks in
-advance (by generating fake proofs, and then setting the difficulty such that
-they appear valid). Generalized, this is a `1/n^t^` chance of successfully
-predicting difficulty `t` adjustment periods (`2t` weeks) in advance. Therefore
-the use of the light relay provides stronger security properties to stateless
-SPV proofs when used as an additional validation step, as even entities with
-significant mining resources have a greatly reduced chance of creating fake
-proofs.
+Knowing the current difficulty gives basic stateless SPV proofs additional,
+stronger recency assurances. Any newly-generated stateless SPV must include that
+difficulty in its header chain, and that difficulty is not known to any party in
+advance. Miners with an `n`-fraction (as usual, `n >= 2` due to the 51%
+assumption) of the hashrate have a `1/n` chance of being allowed to set the
+difficulty, and thus have a `1/n` chance of being able to successfully predict
+it 2 weeks in advance (by generating fake proofs, and then setting the
+difficulty such that they appear valid). Generalized, this is a `1/n^t^` chance
+of successfully predicting difficulty `t` adjustment periods (`2t` weeks) in
+advance. Therefore the use of the light relay provides stronger security
+properties to stateless SPV proofs when used as an additional validation step,
+as even entities with significant mining resources have a greatly reduced chance
+of creating fake proofs.
 
 == Lots
 
-Deposits will be managed in fixed-size _lots_. Each deposit therefore will
-have to be of the same amount: {btc-lot-size}. Thus, a depositor submitting
-their <<Proof of deposit,proof of deposit>> must prove that they deposited
-{btc-lot-size} into the deposit's signing group wallet. If a depositor wants to
-deposit more than the lot size, they will need to create multiple deposit
-requests and fund multiple deposits. This allows each deposit to be backed by
-a different signing group, both simplifying the bonding of signing groups and
-improving the resilience of the system to signing group failure, malicious or not.
+When creating a deposit, the deposit's _lot size_ must be specified. The tBTC
+system supports a governable set of available lot sizes, guaranteeing only that
+at least 1 BTC lot sizes will be available. v1 will launch with four available
+lot sizes: 0.002 BTC, 0.01 BTC, 0.1 BTC, 0.2 BTC, 0.5 BTC, and 1 BTC. Deposit
```

Comment:
This is 6 lot sizes, not 4. Either way I'd have this link to a stub governance section since it's the second handwavy mention I've seen

Reply:
I guess we can link to governance now @shadowfiend

Reply:
Oh suuuuuure :wink: